### ISSUE TITLE
Call to undefined method Illuminate\Foundation\Bootstrap\ConfigureLogging::configureHandler

### ISSUE THREAD

**Issue by xbagir** [laravel/framework#7629]
When run "composer update ", got error:

Fatal error: Call to undefined method Illuminate\Foundation\Bootstrap\ConfigureLogging::configureHandler() in ...\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\ConfigureLogging.php on line 54
Script php artisan clear-compiled handling the post-update-cmd event returned with an error

**Comment by GrahamCampbell** [comment]
Delete the compiled file by hand.

**Comment by xbagir** [comment]
Deleting will not help.

The error in the method:

```php
protected function configureHandlers(Application $app, Writer $log)
	{
		$method = "configure".ucfirst($app['config']['app.log'])."Handler";

		$this->{$method}($app, $log);
	}
```
It seems to do so
```php
$method = "configure".ucfirst($app['config']['app.log'])."Handlers";
```

**Comment by DragonI** [comment]
error messages when updating from 5.0.7 to 5.0.8

>Generating autoload files

>Notice: date_default_timezone_set(): Timezone ID '' is invalid in D:\www\Larave
>l\test\L5-test\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\
>LoadConfiguration.php on line 41

>Fatal error: Call to undefined method Illuminate\Foundation\Bootstrap\ConfigureL
>ogging::configureHandler() in D:\www\Laravel\test\L5-test\vendor\laravel\fr
>amework\src\Illuminate\Foundation\Bootstrap\ConfigureLogging.php on line 54
>Script php artisan clear-compiled handling the post-update-cmd event returned wi
>th an error

**app.php**
'timezone' => 'UTC'

**Comment by RomainLanz** [comment]
Same here : https://laracasts.com/discuss/channels/general-discussion/l5-date-default-timezone-error-on-clear-compiled/replies/38236
And here : https://laracasts.com/discuss/channels/general-discussion/laravel-5-composer-update-error-on-fresh-installation

**Comment by RomainLanz** [comment]
Fixed?

https://github.com/laravel/framework/pull/7632

**Comment by DragonI** [comment]
Sorry, same error when upgrading to 5.0.11

**Comment by RomainLanz** [comment]
And with 5.0.10?

**Comment by DragonI** [comment]
Yes, same errors with 5.0.10 as well. Reverted back to 5.0.7 and it's all good

**Comment by RomainLanz** [comment]
Wait for https://github.com/laravel/framework/pull/7634

**Comment by RomainLanz** [comment]
Correct here https://github.com/laravel/framework/commit/839c2f9a9f07948037f76be647a0131fbdc465a0

Can you confirm?

**Comment by DragonI** [comment]
5.0.12 worked like a charm :) Thx @RomainLanz

**Comment by RomainLanz** [comment]
@Dragonl thanks @portonefive :)

**Comment by portonefive** [comment]
See #7635 for the proper fix.

**Comment by KrisRoofe** [comment]
When I upgrade my laravel to 5.0 from 4.2, I copy my app.php.
I solved it by this [#6310](https://github.com/laravel/framework/issues/6310)
